### PR TITLE
fix: update ntp role to include all private IPs

### PR DESCRIPTION
**Pull request by surajsbharadwaj**

### CODE REVIEW

File: roles/powervs_client_enable_services/tasks/main.yml [hunk 1]
```diff
@@ -51,6 +51,9 @@
       when:
         - client_config.squid.no_proxy_hosts is defined
         - not replace_noproxy_log.changed
+    - name: Check that you can connect (GET) to a page and it returns a status 200
```
Comment (surajsbharadwaj) [inline roles/powervs_client_enable_services/tasks/main.yml:54]: Test if internet is accessible and a 200 status is returned

File: roles/powervs_client_enable_services/tasks/main.yml [hunk 2]
```diff
@@ -69,7 +77,7 @@
         name: chrony
         state: present
 
-    ## modifying /etc/chronyd file
+    ## Modifying /etc/chronyd file
     - name: Replace iburst line in /etc/chronyd file
```
Comment (surajsbharadwaj) [inline roles/powervs_client_enable_services/tasks/main.yml:81]: Replace iburst line in /etc/chrony.conf file

File: roles/powervs_install_services/tasks/main.yml [hunk 3]
```diff
@@ -96,17 +96,24 @@
         state: latest
 
     # modify /etc/chronyd file
-    - name: Replace iburst line in /etc/chronyd file
+    - name: Replace iburst line in /etc/chrony.conf file
```
Comment (surajsbharadwaj) [inline roles/powervs_install_services/tasks/main.yml:99]: Replace startum may be ?

File: roles/powervs_install_services/tasks/main.yml [hunk 4]
```diff
@@ -96,17 +96,24 @@
...
       replace:
         path: /etc/chrony.conf
         regexp: "^(.*)local stratum 10$"
         replace: "local stratum 10"
 
-    - name: Edit allowed subnet in /etc/chronyd file
+    - name: Edit allowed subnets in /etc/chrony.conf file
```
Comment (surajsbharadwaj) [inline roles/powervs_install_services/tasks/main.yml:105]: Add all private IPv4 addresses in allowed list in /etc/chrony.conf file

File: roles/powervs_client_enable_services/tasks/main.yml (hunk 1 above)
Comment (sahityajain123) [reply roles/powervs_client_enable_services/tasks/main.yml]: corrected

File: roles/powervs_client_enable_services/tasks/main.yml (hunk 2 above)
Comment (sahityajain123) [reply roles/powervs_client_enable_services/tasks/main.yml]: Corrected

File: roles/powervs_install_services/tasks/main.yml (hunk 4 above)
Comment (surajsbharadwaj) [reply roles/powervs_install_services/tasks/main.yml:105]: Please fix this as well

File: roles/powervs_install_services/tasks/main.yml (hunk 3 above)
Comment (surajsbharadwaj) [reply roles/powervs_install_services/tasks/main.yml:99]: and this

File: roles/powervs_install_services/tasks/main.yml (hunk 4 above)
Comment (sahityajain123) [reply roles/powervs_install_services/tasks/main.yml:105]: fixed

File: roles/powervs_install_services/tasks/main.yml (hunk 3 above)
Comment (surajsbharadwaj) [reply roles/powervs_install_services/tasks/main.yml]: @sahityajain123 there are 2 tasks with same name. May this can have different name?

**Pull request closed without merging**